openshift/template.yaml

```diff
@@ -287,16 +207,6 @@ parameters:
   value: "false"
 - name: PROW_JOBS_SCRAPER_SCHEDULE
   value: "@hourly"
-- name: EQUINIX_USAGES_SCRAPE_INTERVAL
```

Comment:
if we only want to temporarily pause data collection and costs report, maybe we can instead "feature-flag" this thing?
because bringing back all this code will be harder to reintroduce with all the git conflicts. a few boolean flags will be easier to re-toggle